activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-api/src/main/java/org/activiti/cloud/services/rest/api/ProcessInstanceController.java

```diff
@@ -38,6 +39,9 @@
     @RequestMapping(value = "/signal")
     ResponseEntity<Void> sendSignal(@RequestBody SignalProcessInstancesCmd cmd);
 
+    @RequestMapping(value = "/broadcast-signal")
```

Comment:
Remove this endpoint because this function can be include in the sendSignal endpoint.